### DISCUSSION TITLE
Spoke isn't Accessible from Localhost

### DISCUSSION THREAD

**Discussion by redhatpanda** [mozilla/hubs#5310] [Spoke Questions]
**Description**
I've cloned the repo into my local machine, and tried executing it. 
Executing :
```
npm run dev
```
this, spins up the local server. Although, when I try to access Spoke, it is redirecting to the homepage itself. Am I missing something while setting up the project before executing it?

Please refer to this video :
https://user-images.githubusercontent.com/67406959/160603585-2698aa1c-7a08-4799-9a6d-025d4eb84a3b.mp4

**Comment by matthewbcool** [comment]
Check out this documentation to run spoke locally: [Spoke dev docs](https://github.com/mozilla/Spoke/blob/master/docs/README.md#getting-started)

For running modified spoke on hubs cloud: https://discord.com/channels/498741086295031808/819203055945383986/953811123352764476
